src/iomusxml.cpp

```diff
@@ -2639,6 +2639,7 @@ void MusicXmlInput::ReadMusicXmlNote(
             else {
                 MRest *mRest = new MRest();
                 element = mRest;
+                mRest->SetDurPpq(duration);
```

Comment:
This will not turn up in the MEI file. You have to add i/o for this to `iomei.cpp`.